tweeli/config.ini

```diff
@@ -0,0 +1,10 @@
+[Auth]
```

Comment:
also at this stage I don't think it's critical for us to change the naming scheme! We will stay with calling it API since it's for API Usage at the end 😉